src/styles/common.scss

```
@@ -1,12 +1,16 @@
 *,
 html,
 body {
-  font-family: "Spoqa Han Sans Neo", "sans-serif";
+  font-family: 'Spoqa Han Sans Neo', 'sans-serif';
 }
 
 // float 초기화
 .clear::after {
-  content: "";
+  content: '';
   display: block;
   clear: both;
 }
+
+*.strong {
```

```suggestion
.strong {
```
전체 선택자 없이도 사용 가능하지 않을까요?

---

확인하여 적용되었습니다.